json/src/test/scala/com/gu/contentapi/json/CirceSpec.scala

```diff
@@ -1,92 +0,0 @@
-package com.gu.contentapi.json
-
-import com.gu.contentapi.json.utils.JsonLoader
-import com.gu.contentatom.thrift.Atom
-import com.gu.contentapi.client.model.v1._
-import com.gu.contentapi.json.CirceSerialization._
-
-import io.circe._
-import io.circe.parser._
-
-import org.json4s.jackson.JsonMethods
-
-import cats.data.Xor.Right
-
-import org.scalatest.{FlatSpec, Matchers}
-
-class CirceSpec extends FlatSpec with Matchers {
-
-  def compareAgainstJson4s[T: Decoder: Manifest](filename: String) = {
-    val jsonString = JsonLoader.loadJson(filename)
-
-    val usingCirce = {
-      val json: Json = parse(jsonString).getOrElse(Json.Null)
-      json.as[T]
-    }
-
-    val usingJson4s = {
-      import org.json4s._
-      implicit val formats = Serialization.formats + new CustomSerializer[Boolean](format => (
-        {
-          case JBool(value) => value
-          case JString("true") => true
-          case JString("false") => false
-        },
-        {
-          case true => JBool(true)
-          case false => JBool(false)
-        }
-        ))
-      val json: JValue = JsonMethods.parse(jsonString)
-      json.extract[T]
-    }
-
-    usingCirce should be(Right(usingJson4s))
-  }
-
-  it should "be able to deserialize a Json number to a String" in {
```

Comment:
This test and the next one are worth keeping. They don't do any comparison against json4s.